src/components/pages/CreateStandardPage.js

```diff
@@ -0,0 +1,281 @@
+import { React, useContext, useState } from "react";
+import Navbar from "../reusable/NavBar";
+import Sidebar from "../reusable/SideBar";
+import StockSelect from "../singular/StockSelect";
+import AlgoquantApiContext from "../../api/ApiContext";
+import IndicatorSelect from "../singular/IndicatorSelect";
+import { useNavigate } from "react-router-dom";
+
+const CreateStandardPage = () => {
+  const navigate = useNavigate();
+
+  // State variables used to access algoquant SDK API and display/ keep state of user data from database
+  const algoquantApi = useContext(AlgoquantApiContext);
+
+  // State variables used to keep track of user input
+  const [investorName, setInvestorName] = useState(null);
+  const [searchResults, setSearchResults] = useState([]);
+  const [tradeFrequency, setTradeFrequency] = useState("Select");
+  const [profitStop, setProfitStop] = useState(50);
+  const [lossStop, setLossStop] = useState(50);
+  const [selectedIndicators, setSelectedIndicators] = useState([]);
+  const [selectedStocks, setSelectedStocks] = useState([]);
+
+  // State variables used to display error messages to the user
+  const [nameError, setNameError] = useState(false);
+  const [tradeFrequencyError, setTradeFrequencyError] = useState(false);
+  const [indicatorError, setIndicatorError] = useState(false);
+  const [stocksError, setStocksError] = useState(false);
+
+  // Function called anytime a user selects an indicator
+  const handleIndicatorSelect = (selectedOptions) => {
+    setSelectedIndicators(selectedOptions);
+  };
+
+  // Function called anytime a user selects a stock
+  const handleStockSelect = (stocks) => {
+    setSelectedStocks(stocks);
+  };
+
+  /*
+  Function called when the user attempts to save changes. Will check all the user values and 
+  attempt to update them.
+  */
+  const saveChanges = () => {
```

Comment:
A user is able to select no stocks, a user is required to input at least one stock.

Reply:
Not sure what you mean by this

Reply:
@ryPattillo

Reply:
You don't have any checks for if a user has slelected any stocks. Therefore , a user should be required to select at least one stock to create an investor

Reply:
Nah it's in there, the if statement was checking for indicators instead. i just fixed it